code/modules/cargo/packs/gun.dm

```diff
@@ -115,6 +115,14 @@
 					/obj/item/gun/ballistic/shotgun/flamingarrow)
 	crate_name = "rifle crate"
 
+/datum/supply_pack/gun/cobra20
+	name = "Cobra-20 SMG Crate"
+	desc = "Contains two .45 submachine guns, manufactured by Scaraborough Arms and chambered in .45"
+	cost = 2500
```

Comment:
I'd bump this up to 3,000 credits. Having only a 500 credit gap between this and a crate of pistols does mean there's very little reason to ever buy the pistols when you're so close to SMGs, and it leaves enough of a gap between the SMG crate and Illestrens or shotguns to fill a useful gap. 2500's also exactly the same as the revolver crates, and it should probably cost a little more than those.